pkg/deploy/assets/rp-production.json

```diff
@@ -541,7 +541,7 @@
             },
             "location": "[resourceGroup().location]",
             "name": "[substring(parameters('storageAccountDomain'), 0, indexOf(parameters('storageAccountDomain'), '.'))]",
-            "type": "Microsoft.Storage/storageAccounts",
+            "type": "Microsoft.Storage/storageAccounts", { "properties": "allowSharedAccessKey: false" },
```

Comment:
Note to self: I can add `\n` to make this better indented. This was just a proof of concept / draft.